lib/claude/process-manager.ts

```
@@ -0,0 +1,450 @@
+  async sendMessage(sessionId: string, content: string): Promise<void> {
+    const session = this.processes.get(sessionId);
+    if (!session) {
+      throw new Error(`Session ${sessionId} not found`);
+    }
+
+    if (session.status !== 'active') {
+      throw new Error(`Session ${sessionId} is not active`);
+    }
+
+    try {
+      session.claudeProcess.stdin?.write(content + '\n');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)